Enhancing Fusion Docs (#7833)
fishtown-analytics/docs.getdbt.com: website/snippets/_install-dbt-extension.md

## Troubleshooting
<!-- This anchor is linked from the  VS Code extension. Please do not change it -->

#### dbt platform configurations

Users of the cloud-based dbt platform who have a `dbt-cloud:` config in the `dbt_project.yml` file and who are also using [dbt Mesh](/docs/mesh/about-mesh) must have the project ID configured:

```yaml
dbt-cloud:
  project-id: 12345 # Required
```

> 

Author: ```suggestion

Otherwise, cross-platform references will not resolve, and you are likely to encounter errors executing dbt commands
```

## Troubleshooting
<!-- This anchor is linked from the  VS Code extension. Please do not change it -->

#### dbt platform configurations

> Users of the cloud-based dbt platform who have a `dbt-cloud:` config in the `dbt_project.yml` file and who are also using [dbt Mesh](/docs/mesh/about-mesh) must have the project ID configured:

Author: ```suggestion
If you're a cloud-based dbt platform user who has the `dbt-cloud:` config in the `dbt_project.yml` file and are also using [dbt Mesh](/docs/mesh/about-mesh), you must have the project ID configured:
```

```yaml
dbt-cloud:
  project-id: 12345 # Required
```

> Otherwise, cross-platform references will not resolve, and you are likely to encounter errors executing dbt commands

Author: ```suggestion
Otherwise, cross-platform references will not resolve, and you are likely to encounter errors executing dbt commands.

```

## Troubleshooting
<!-- This anchor is linked from the  VS Code extension. Please do not change it -->

#### dbt platform configurations

If you're a cloud-based dbt platform user who has the `dbt-cloud:` config in the `dbt_project.yml` file and are also using [dbt Mesh](/docs/mesh/about-mesh), you must have the project ID configured:

```yaml
dbt-cloud:
  project-id: 12345 # Required
```

> Otherwise, cross-platform references will not resolve, and you are likely to encounter errors executing dbt commands.

Author: ```suggestion
Otherwise, cross-platform references will not resolve properly, and you will encounter errors executing dbt commands.
```

> Otherwise, cross-platform references will not resolve properly, and you will encounter errors executing dbt commands.

Reviewer: ```suggestion
If you don’t configure this correctly, cross-platform references will not resolve properly, and you will encounter errors executing dbt commands.
```
